gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.8.1-all.zip
```

Comment:
The correct way to update Gradle wrapper is by running `wrapper` task. It'll update the wrapper jar as well. Like this:
```
./gradlew wrapper --distribution-type=all --gradle-version=4.8.1
```